.github/workflows/docker-build-test.yaml

```diff
@@ -130,6 +130,23 @@ jobs:
   # This is a PR required job.
   rust-images:
     needs: [permission-check, determine-docker-build-metadata]
+    runs-on: ubuntu-latest
```

Comment:
Do not put this here, this is really hard to read and even harder to test, please add it to some existing script